client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/TablesApiClientFactory.java

```diff
@@ -50,6 +50,7 @@ public ApiClient createApiClient(@NonNull String baseUrl, String token, String t
       throws MalformedURLException, SSLException {
     WebClient webClient = createWebClient(baseUrl, token, truststoreLocation);
     ApiClient apiClient = new ApiClient(webClient);
+    apiClient.addDefaultHeader("Authorization", String.format("Bearer %s", token));
```

Comment:
Was the secure client not used before in local docker, and hence it worked without the token being set?

Reply:
@abhisheknath2011 what is the significance of Secure client, certs-only? What do we use in local docker?

Reply:
@sumedhsakdeo Secure client provides way to create one way TLS client and manages HTTP connection pool on the client side. Another significance of Secure client is to provide common way to create REST clients to interact with openhouse. Openhouse catalog and and other parts of the codebase within openhouse/li-openhouse uses Secure client for interacting with openhouse services. In case of local docker secure client is used internally when we interact with openhouse catalog.